docs: align landing pages with model (#2272)
canonical/charmcraft: docs/tutorial/index.rst

Tutorial
========

> Charmcraft's tutorials are the best entry-point for anyone new to the charming world.
> These tutorials will help you learn the basics of charming from functional, reproducible
> examples.

Reviewer: ```suggestion
The Charmcraft tutorials are the best entry point for anyone new to making charms.
They teach the basics of charming from functional, reproducible examples.
```

Author: https://github.com/canonical/charmcraft/pull/2272/commits/a200349c7334c7deaa19accd24d4e959a17cf41a

12-factor app extensions
------------------------

Charmcraft's extensions enable you to build Django, FastAPI, Flask, and Go applications
with just a few modifications to the YAML template content. To familiarize yourself

> with these workflows, check out the following tutorials:

Reviewer: ```suggestion
with one of these workflows, start with its tutorial:
```

Author: https://github.com/canonical/charmcraft/pull/2272/commits/a200349c7334c7deaa19accd24d4e959a17cf41a
